### ISSUE TITLE
Park label hide sculpture

### ISSUE THREAD

**Issue by ivanbarsukov** [organicmaps/organicmaps#4747]
**Steps to reproduce**
1. Go to https://omaps.app/4z7Gq8F2Q8/%D1%81%D0%BA%D0%B2%D0%B5%D1%80_%D0%B8%D0%BC%D0%B5%D0%BD%D0%B8_%D0%A2%D0%B0%D1%82%D1%82%D1%8B%D0%B1%D1%8E%D0%B1%D1%8E_%D0%A2%D1%83%D1%80%D1%81%D1%83%D0%BD%D0%B1%D0%B0%D0%B5%D0%B2%D0%BE%D0%B9
2. There are no sculptures, but https://www.openstreetmap.org/node/5141873535

**Expected behavior**
Sculptures are visible at close zoom

**Screenshots**
![image](https://user-images.githubusercontent.com/14816075/224214816-508e918c-6b2e-4f05-b682-60ebf7493088.png)

**Comment by biodranik** [comment]
Drawing the park's name is more important than drawing a sculpture, no?

**Comment by ivanbarsukov** [comment]
> Drawing the park's name is more important than drawing a sculpture, no?

Of course, it is very important, but in my opinion, this is more important with survey zooms. And with a close zooms, as a rule, you already know where you are and you are interested in physical objects. Plus, on those zooms, the park name (if the park is big enough) will not always be in viewport, so in this case a more convenient option is to long press. Probably this is more general issue, but this situation is representative

![image](https://user-images.githubusercontent.com/14816075/224525403-d10d9de9-d5aa-4384-ad13-6916149c7e51.png)

**Comment by biodranik** [comment]
What is the behavior of Google and Apple maps for similar cases? Do they stop showing the large park name on the detailed zoom?

**Comment by ivanbarsukov** [comment]
It's hard to find similar case with overlapping because of different geometries of park polygon and maybe special point for showing name. Also apple maps probably even not showing sculptures. But label are showing till the closest zoom. My point not in stop showing park name but in priorities on the detailed zoom. It's fine to see park name if it doesn't overlaps POIs on this zoom.

I think it could be a frequent problem because in OM I found next example in 13 minutes by foot

![image](https://user-images.githubusercontent.com/14816075/231934462-e0771075-eaf0-4a88-b4cb-8e66bf2d5aca.png)
![image](https://user-images.githubusercontent.com/14816075/231934490-b6d90496-880b-43fb-93c8-919c3fab415d.png)
![image](https://user-images.githubusercontent.com/14816075/231934499-447972b9-7239-4e37-ad50-c822f9601e46.png)

**Comment by pastk** [comment]
We've been discussing this logic before (https://github.com/organicmaps/organicmaps/issues/4314#issuecomment-1426645820) on some other examples, e.g. when zooming in
- shopping mall icon getting replaced by specific shops
- airport terminal replaced with amenities inside
- city or neighborhood label replaced with local POIs (atm city labels just disappear at some zoom and for some cases the label is visible for 2-3 zoom levels only, which is not enough)

It shoud be possible to use different priorities for different zoom levels, e.g. define `leisure-park::survey_zooms` and `leisure-park::detailed_zooms`. Maybe it'll work well in combination with `[bbox>X]` even to adjust for different area sizes. But maybe it'll be OK even without it - needs some experimentation.

**Comment by patepelo** [comment]
related to #6332
